masterlist.yaml

```diff
@@ -270,11 +270,20 @@ plugins:
       - 'DLCCoast.esm'
       - 'DLCworkshop02.esm'
       - 'DLCworkshop03.esm'
+  - name: 'DLCUltraHighResolution.esm'
+    global_priority: -120
+    after:
+      - 'DLCRobot.esm'
+      - 'DLCworkshop01.esm'
+      - 'DLCCoast.esm'
+      - 'DLCworkshop02.esm'
+      - 'DLCworkshop03.esm'
+      - 'DLCNukaWorld.esm'
```

Comment:
As @WrinklyNinja said, perhaps leave this out for now until we figure out what to do with it? There's also already another PR already just for this specific change: #34